src/test/resources/config-unit-test.yml

```diff
@@ -1,3 +1,11 @@
+server:
```

Comment:
Why is this needed? The `AppTest` did not change. I guess the default is to start on ports 8080 and 8081? Maybe these should actually change to zero to select random ports? Which assumes `DropwizardAppExtension` provides a way to get the assigned ports, which I cannot remember at the moment.

Also, `AppTest` only asserts that `UserResource` is registered. It should probably be updated to check the other resources too.

Reply:
I added it because I had the service running locally and tried to run the test and it failed because of the port conflict, I was just trying to allow both to run at the same time.  If there is a way to have it pick a random port then I'm all for it.

Reply:
I _think_ you can set the ports to zero and it will use a random port.